Update provisining guide with updated Azure info and Gov Cloud (#546)
theforeman/foreman-documentation: guides/common/modules/proc_adding-azure-details-to-a-compute-profile.adoc

. Optional: In the *Custom Script Command* field, enter commands to perform on the virtual machine when the virtual machine is provisioned.
. Optional: If you want to run custom scripts when provisioning finishes, in the *Comma separated file URIs* field, enter comma-separated file URIs of scripts to use.
The scripts must contain `sudo` at the beginning because {ProjectName} downloads files to the `/var/lib/waagent/custom-script/download/0/` directory on the host and scripts require sudo privileges to be executed.

> . If you want to add an *NVIDIA Driver* you can as well by selecting the *NVIDIA driver / CUDA* checkbox. 

Reviewer: ```suggestion
. (Optional) Add an *NVIDIA Driver* by selecting the *NVIDIA driver / CUDA* checkbox. 
```

Reviewer 2: `Optional:` would be in-line with IBM Style

Reviewer: my bad!

>   For more information, refer to the following Microsoft Azure documentation:

Reviewer: ```suggestion
For more information, refer to the following Microsoft Azure documentation:
```

>   - https://docs.microsoft.com/en-us/azure/virtual-machines/extensions/hpccompute-gpu-linux[NVIDIA GPU Driver Extension for Linux]

Reviewer: ```suggestion
** https://docs.microsoft.com/en-us/azure/virtual-machines/extensions/hpccompute-gpu-linux[NVIDIA GPU Driver Extension for Linux]
```
Same for the other links.

Reviewer 2: I don't understand the value of `**`?

Reviewer: It indicates that the unordered list item is on a lower level than the ordered list:

. item 1
. item 2
** first bullet point for item2
** second bullet point for item2
. item 3

Reviewer 2: But he's using an ordered list, with `.` so you'd need a first need a layer of an ordered list?

Reviewer: Yes, going with `*` is also perfectly fine for asciidoc.

This is just for convencience, as it directly shows the "nested" aspect of the two lists.

  - https://docs.microsoft.com/en-us/azure/virtual-machines/extensions/hpccompute-gpu-windows[NVIDIA GPU Driver Extension for Windows]
. Optional: If you want to create an additional volume on the VM, click the *Add Volume* button, enter the *Size* in GB and select the *Data Disk Caching* method.

>   - Note that the maximum number of these disks depends on the VM Size selected. For more information on Microsoft Azure VM storage requirements, see the https://docs.microsoft.com/en-us/azure/virtual-machines[Microsoft Azure documentation].

Reviewer: ```suggestion
** Note that the maximum number of these disks depends on the VM Size selected.
For more information on Microsoft Azure VM storage requirements, see the https://docs.microsoft.com/en-us/azure/virtual-machines[Microsoft Azure documentation].
```

. Click *Add Interface*.

>   - *IMPORTANT*: The maximum number of interfaces depends on the VM Size selected.

Reviewer: ```suggestion
+
[IMPORTANT]
====
The maximum number of interfaces depends on the VM Size selected.
For more information, see the Microsoft Azure documentation link above.
====
```

. Optional: In the *Custom Script Command* field, enter commands to perform on the virtual machine when the virtual machine is provisioned.
. Optional: If you want to run custom scripts when provisioning finishes, in the *Comma separated file URIs* field, enter comma-separated file URIs of scripts to use.
The scripts must contain `sudo` at the beginning because {ProjectName} downloads files to the `/var/lib/waagent/custom-script/download/0/` directory on the host and scripts require sudo privileges to be executed.

> . If you want to add an *NVIDIA Driver* you can as well by selecting the *NVIDIA driver / CUDA* checkbox. Please refer to the Microsoft documentation links below for more details.

Reviewer: ```suggestion
. If you want to add an *NVIDIA Driver* you can as well by selecting the *NVIDIA driver / CUDA* checkbox.
Please refer to the Microsoft documentation links below for more details.
```

> . If you want to add an *NVIDIA Driver* you can as well by selecting the *NVIDIA driver / CUDA* checkbox. Please refer to the Microsoft documentation links below for more details.

Reviewer 2: ```suggestion
. If you want to add an *NVIDIA Driver* you can as well by selecting the *NVIDIA driver / CUDA* checkbox. For more information, refer to the following Microsoft Azure documentation:
```

  - https://docs.microsoft.com/en-us/azure/virtual-machines/extensions/hpccompute-gpu-linux[NVIDIA GPU Driver Extension for Linux]
  - https://docs.microsoft.com/en-us/azure/virtual-machines/extensions/hpccompute-gpu-windows[NVIDIA GPU Driver Extension for Windows]
. Optional: If you want to create an additional volume on the VM, click the *Add Volume* button, enter the *Size* in GB and select the *Data Disk Caching* method.

>   - *IMPORTANT*: Note that the maximum number of these disks depends on the VM Size selected. Please refer to the Microsoft documentation links below for more details.

Reviewer 2: ```suggestion
 Note that the maximum number of these disks depends on the VM Size selected. For more information on Microsoft Azure VM storage requirements, see the Microsoft Azure documentation.
```

>     - https://docs.microsoft.com/en-us/azure/virtual-machines/sizes-general[General Purpose]

Reviewer 2: Perhaps just provide one overall VM link, rather than all these. 
These links might break in time. I'd literally just include this one https://docs.microsoft.com/en-us/azure/virtual-machines and tell users to go look there.

    - https://docs.microsoft.com/en-us/azure/virtual-machines/sizes-compute[Compute Optimized]
    - https://docs.microsoft.com/en-us/azure/virtual-machines/sizes-memory[Memory Optimized]
    - https://docs.microsoft.com/en-us/azure/virtual-machines/sizes-storage[Storage Optimized]
    - https://docs.microsoft.com/en-us/azure/virtual-machines/sizes-gpu[GPU Optimized]
    - https://docs.microsoft.com/en-us/azure/virtual-machines/sizes-hpc[High Performance]

. Click *Add Interface*.

>   - *IMPORTANT*: The maximum number of interfaces depends on the VM Size selected. Please refer to the Microsoft documentation links above for more details.

Reviewer 2: ```suggestion
The maximum number of interfaces depends on the VM Size selected. For more information, see the Microsoft Azure documentation. 
```

IBM says never say "please" ;-)

. Optional: In the *Custom Script Command* field, enter commands to perform on the virtual machine when the virtual machine is provisioned.
. Optional: If you want to run custom scripts when provisioning finishes, in the *Comma separated file URIs* field, enter comma-separated file URIs of scripts to use.
The scripts must contain `sudo` at the beginning because {ProjectName} downloads files to the `/var/lib/waagent/custom-script/download/0/` directory on the host and scripts require sudo privileges to be executed.

> . Optional: Add an *NVIDIA Driver* you can as well by selecting the *NVIDIA driver / CUDA* checkbox. 

Reviewer 2: ```suggestion
. Optional: You can add a *NVIDIA Driver* by selecting the *NVIDIA driver / CUDA* checkbox. 
```
